pkg/sql/logictest/testdata/logic_test/row_level_ttl

```diff
@@ -1251,6 +1251,7 @@ query TTT
 SELECT schedule_status, recurrence, owner FROM [SHOW SCHEDULES]
 WHERE label = 'row-level-ttl: $label_suffix'
 ----
+ACTIVE  @hourly  root
 
 subtest end
```

Comment:
Can we add one more test case here (under subtest `subtest set_both_reset_ttl_expire_after`) where the `ttl_expirataion_expression` does *not* use the `expire_at` column? We should expect such stmt to succeed, and expect to see that `reset (ttl_expire_after)` drop the `expire_at` column, modify the TTL (but not removing it altogether)?

Reply:
`ttl_expire_after` uses `crdb_internal_expiration` so `reset (ttl_expire_after)` drops `crdb_internal_expiration` not `expire_at`.